tests/layer_tests/pytorch_tests/test_norm.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+from pytorch_layer_test_class import PytorchLayerTest
+import numpy as np
+import torch
+
+
+@pytest.mark.parametrize('p', [1, 2])
```

Comment:
I think we can support other norms too using formulas from https://pytorch.org/docs/stable/generated/torch.linalg.norm.html#torch.linalg.norm. Maybe not at best performance though.

Reply:
I also thought about it, but I decided to use the following approach:
https://github.com/pytorch/pytorch/blob/master/torch/onnx/symbolic_opset9.py#L3316:~:text=feature_dropout-,%40,),-%40
But I can broaden my implementation to the case, where `p` is different than 1 or 2.

Reply:
Yes, please support other norms, we do not need to restrict ourselves with onnx conversion

Reply:
All types of numeric norms are supported. Frobenius and nuclear norms are separate operators - namely `aten::nuclear_norm` and `aten::frobenius_norm`.